code/src/summary.css

```diff
@@ -0,0 +1,8 @@
+.summary {
+  background-color: #8b95c9ff;
+  color: white;
+  margin-top: 50px;
+  width: 50%;
+  padding: 25px;
+  text-align: center;
```

Comment:
Text that is longer than three lines can be difficult to read if it is centered. Try to think like books, they start from the margin on the left. Just a tip from a UX Writer. :)